### PR TITLE
Docker cleanup

### LINKED ISSUE #726: docker testnet support
Hi,
I'm trying to start a testnet standalone docker instance and I've noticed dockerfile only checks for mocknet otherwise starts on mainnet.

Is it possible to add support for testnet? Is it already available in some other way I'm not seeing?

>RUN printf '#!/bin/bash\n\
MAINNET_ID=0x00000001\n\
MOCKNET_ID=0x80000000\n\
[ $1 = "mocknet" ] && STACKS_CHAIN_ID="${MOCKNET_ID}" || STACKS_CHAIN_ID="${MAINNET_ID}"\n\
export STACKS_CHAIN_ID\n\
trap "exit" INT TERM\n\
trap "kill 0" EXIT\n\
echo Your container args are: "$@"\n\
tail --retry -F stacks-api.log stacks-node.log 2>&1 &\n\
while true\n\
do\n\
  pg_start\n\
  stacks_api &> stacks-api.log &\n\
  stacks_api_pid=$!\n\
  if [ $1 = "mocknet" ]; then\n\
    stacks-node start --config=/home/stacky/Stacks-mocknet.toml &> stacks-node.log &\n\
  else\n\
    stacks-node mainnet &> stacks-node.log &\n\
  fi\n\

### PR DESCRIPTION
**Pull request by rafaelcr**
## Description

This patch cleans up our Dockerfiles, updates the Stacks node version, and organizes dependencies in a `docker/` subdirectory.

Originally authored by @wileyj 

Closes #726 #588 #865 

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] API reference/documentation update
- [x] Other

## Does this introduce a breaking change?
No

## Are documentation updates required?
No

## Testing information
Local testing

## Checklist
- [ ] Code is commented where needed
- [ ] Unit test coverage for new or modified code paths
- [ ] `npm run test` passes
- [ ] Changelog is updated
- [x] Tag 1 of @kyranjamie or @zone117x for review

### CODE REVIEW

**Comment by wileyj** [comment]
Awesome! thanks for the assist @rafaelcr

File: docker/stx-rosetta.Dockerfile [hunk 1]
```diff
@@ -1,15 +1,20 @@
-ARG STACKS_API_VERSION=v0.63.0
-ARG STACKS_NODE_VERSION=2.0.11.2.0
-ARG STACKS_API_REPO=hirosystems/stacks-blockchain-api
+ARG STACKS_API_VERSION=v0.71.0
```
Comment (CharlieC3) [inline docker/stx-rosetta.Dockerfile:1]: Small nit
```suggestion
ARG STACKS_API_VERSION=v0.71.2
```

File: stacks-blockchain/Stacks-mainnet.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+[burnchain]
+chain = "bitcoin"
+mode = "mainnet"
+peer_host = "bitcoin.blockstack.com"
+username = "blockstack"
+password = "blockstacksystem"
+rpc_port = 8332
+peer_port = 8333
+magic_byte = "X2"
```
Comment (CharlieC3) [inline stacks-blockchain/Stacks-mainnet.toml:15]: This can be deleted since it's automatically set in the stacks-node.
```suggestion
```

**Review by CharlieC3 — APPROVED** [review]
Looks great! A couple of small nits, but it's fine to merge without them.

**Review by zone117x — APPROVED** [review]
LGTM!

**Pull request merged**